From 9629e6cc1aa0ca2d04a387efb432d0daee0e4ec4 Mon Sep 17 00:00:00 2001
From: Ernestas Kulik <ekulik@redhat.com>
Date: Wed, 10 Jun 2020 07:03:43 +0200
Subject: [PATCH] Adapt to different exception messages in Java 11

Out-of-bounds exceptions now additionally spit out the length of the
container.
---
 test/outputs/Linux-aarch64/run_test.log.in          | 2 +-
 test/outputs/Linux-aarch64/run_test.log.in.java-1.7 | 2 +-
 test/outputs/Linux-armv7l/run_test.log.in           | 2 +-
 test/outputs/Linux-armv7l/run_test.log.in.java-1.7  | 2 +-
 test/outputs/Linux-ppc/run_test.log.in              | 2 +-
 test/outputs/Linux-ppc/run_test.log.in.java-1.7     | 2 +-
 test/outputs/Linux-ppc64/run_test.log.in            | 2 +-
 test/outputs/Linux-ppc64/run_test.log.in.java-1.7   | 2 +-
 test/outputs/Linux-ppc64le/run_test.log.in          | 2 +-
 test/outputs/Linux-ppc64le/run_test.log.in.java-1.7 | 2 +-
 test/outputs/Linux-s390/run_test.log.in             | 2 +-
 test/outputs/Linux-s390/run_test.log.in.java-1.7    | 2 +-
 test/outputs/Linux-s390x/run_test.log.in            | 2 +-
 test/outputs/Linux-s390x/run_test.log.in.java-1.7   | 2 +-
 test/outputs/run.log.in                             | 2 +-
 test/outputs/run_inner.log.in                       | 2 +-
 test/outputs/run_jar.log.in                         | 2 +-
 test/outputs/run_package.log.in                     | 2 +-
 test/outputs/run_remote.log.in                      | 2 +-
 test/outputs/run_test.log.in                        | 2 +-
 test/outputs/run_test.log.in.java-1.7               | 2 +-
 test/outputs/run_three_times.log.in                 | 2 +-
 test/outputs/run_try_finally.log.in                 | 2 +-
 23 files changed, 23 insertions(+), 23 deletions(-)

diff --git a/test/outputs/Linux-aarch64/run_test.log.in b/test/outputs/Linux-aarch64/run_test.log.in
index dbf5645..aa2b110 100644
--- a/test/outputs/Linux-aarch64/run_test.log.in
+++ b/test/outputs/Linux-aarch64/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-aarch64/run_test.log.in.java-1.7 b/test/outputs/Linux-aarch64/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-aarch64/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-aarch64/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-armv7l/run_test.log.in b/test/outputs/Linux-armv7l/run_test.log.in
index f5a3f65..4999cb2 100644
--- a/test/outputs/Linux-armv7l/run_test.log.in
+++ b/test/outputs/Linux-armv7l/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-armv7l/run_test.log.in.java-1.7 b/test/outputs/Linux-armv7l/run_test.log.in.java-1.7
index 58a4b57..0f033ec 100644
--- a/test/outputs/Linux-armv7l/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-armv7l/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc/run_test.log.in b/test/outputs/Linux-ppc/run_test.log.in
index 6927e4b..a097351 100644
--- a/test/outputs/Linux-ppc/run_test.log.in
+++ b/test/outputs/Linux-ppc/run_test.log.in
@@ -95,7 +95,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc/run_test.log.in.java-1.7 b/test/outputs/Linux-ppc/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-ppc/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-ppc/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc64/run_test.log.in b/test/outputs/Linux-ppc64/run_test.log.in
index f5a3f65..4999cb2 100644
--- a/test/outputs/Linux-ppc64/run_test.log.in
+++ b/test/outputs/Linux-ppc64/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc64/run_test.log.in.java-1.7 b/test/outputs/Linux-ppc64/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-ppc64/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-ppc64/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc64le/run_test.log.in b/test/outputs/Linux-ppc64le/run_test.log.in
index f5a3f65..4999cb2 100644
--- a/test/outputs/Linux-ppc64le/run_test.log.in
+++ b/test/outputs/Linux-ppc64le/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-ppc64le/run_test.log.in.java-1.7 b/test/outputs/Linux-ppc64le/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-ppc64le/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-ppc64le/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-s390/run_test.log.in b/test/outputs/Linux-s390/run_test.log.in
index 8cc10fd..3f6fd8c 100644
--- a/test/outputs/Linux-s390/run_test.log.in
+++ b/test/outputs/Linux-s390/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-s390/run_test.log.in.java-1.7 b/test/outputs/Linux-s390/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-s390/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-s390/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-s390x/run_test.log.in b/test/outputs/Linux-s390x/run_test.log.in
index f5a3f65..4999cb2 100644
--- a/test/outputs/Linux-s390x/run_test.log.in
+++ b/test/outputs/Linux-s390x/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/Linux-s390x/run_test.log.in.java-1.7 b/test/outputs/Linux-s390x/run_test.log.in.java-1.7
index 6019976..906eb6f 100644
--- a/test/outputs/Linux-s390x/run_test.log.in.java-1.7
+++ b/test/outputs/Linux-s390x/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/run.log.in b/test/outputs/run.log.in
index aa8b2c8..cabcfb2 100644
--- a/test/outputs/run.log.in
+++ b/test/outputs/run.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
diff --git a/test/outputs/run_inner.log.in b/test/outputs/run_inner.log.in
index c1ef063..31f5bf9 100644
--- a/test/outputs/run_inner.log.in
+++ b/test/outputs/run_inner.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
diff --git a/test/outputs/run_jar.log.in b/test/outputs/run_jar.log.in
index 85cf311..9d190e0 100644
--- a/test/outputs/run_jar.log.in
+++ b/test/outputs/run_jar.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [jar:file:@CMAKE_BINARY_DIR@/test/JarTest.jar!/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [jar:file:@CMAKE_BINARY_DIR@/test/JarTest.jar!/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [jar:file:@CMAKE_BINARY_DIR@/test/JarTest.jar!/SimpleTest.class]
diff --git a/test/outputs/run_package.log.in b/test/outputs/run_package.log.in
index 7c7e76f..eb9ceb0 100644
--- a/test/outputs/run_package.log.in
+++ b/test/outputs/run_package.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method com.redhat.abrt.test.Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at com.redhat.abrt.test.Test.throwIndexOutOfBoundsException(Test.java:26) [file:@CMAKE_BINARY_DIR@/test/com/redhat/abrt/test/Test.class]
 	at com.redhat.abrt.test.Test.catchIndexOutOfBoundsException(Test.java:49) [file:@CMAKE_BINARY_DIR@/test/com/redhat/abrt/test/Test.class]
 	at com.redhat.abrt.test.Test.throwAndCatchAllExceptions(Test.java:63) [file:@CMAKE_BINARY_DIR@/test/com/redhat/abrt/test/Test.class]
diff --git a/test/outputs/run_remote.log.in b/test/outputs/run_remote.log.in
index aeb7890..31ef171 100644
--- a/test/outputs/run_remote.log.in
+++ b/test/outputs/run_remote.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [jar:http://localhost:54321/JarTest.jar!/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [jar:http://localhost:54321/JarTest.jar!/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [jar:http://localhost:54321/JarTest.jar!/SimpleTest.class]
diff --git a/test/outputs/run_test.log.in b/test/outputs/run_test.log.in
index dbf5645..aa2b110 100644
--- a/test/outputs/run_test.log.in
+++ b/test/outputs/run_test.log.in
@@ -98,7 +98,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/run_test.log.in.java-1.7 b/test/outputs/run_test.log.in.java-1.7
index a0d53d8..d79668c 100644
--- a/test/outputs/run_test.log.in.java-1.7
+++ b/test/outputs/run_test.log.in.java-1.7
@@ -112,7 +112,7 @@ Exception in thread "main" java.net.MalformedURLException: no protocol: @#$%^&ma
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method Test.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at Test.throwIndexOutOfBoundsException(Test.java:266) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.catchIndexOutOfBoundsException(Test.java:347) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.throwAndCatchAllExceptions(Test.java:486) [file:@CMAKE_BINARY_DIR@/test/Test.class]
diff --git a/test/outputs/run_three_times.log.in b/test/outputs/run_three_times.log.in
index aa8b2c8..cabcfb2 100644
--- a/test/outputs/run_three_times.log.in
+++ b/test/outputs/run_three_times.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
diff --git a/test/outputs/run_try_finally.log.in b/test/outputs/run_try_finally.log.in
index 877f755..7340652 100644
--- a/test/outputs/run_try_finally.log.in
+++ b/test/outputs/run_try_finally.log.in
@@ -1,5 +1,5 @@
 Caught exception java.lang.ArrayIndexOutOfBoundsException in method SimpleTest.throwIndexOutOfBoundsException()
-Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 42
+Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 42 out of bounds for length 10
 	at SimpleTest.throwIndexOutOfBoundsException(SimpleTest.java:24) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.catchIndexOutOfBoundsException(SimpleTest.java:47) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
 	at SimpleTest.throwAndCatchAllExceptions(SimpleTest.java:61) [file:@CMAKE_BINARY_DIR@/test/SimpleTest.class]
-- 
2.26.2

